### ISSUE TITLE
Couple issues

### ISSUE THREAD

**Issue by SteveLobdell** [BHughes3388/BAHYouTubeOAuth#5]
Couple things I noticed that would be nice for updating but easily fixed.

1.) The code is only dismissing the web view if access token and refresh token are populated, so if you don't request offline access the web view just hangs because there is no refresh token.  Should probably just dismiss if only access token has a value.
2.) More of a setup issue, but if you just set the redirect URL to a page without a slash in it, YouTube returns it with a slash.  So originally I set my redirect URL to www.stevelobdell.com, but because there's no sub page YouTube returns it as www.stevelobdell.com/ so when you try to match it against www.stevelobdell.com? It doesnt because the URL is now www.stevelobdell.com/?.
3.) I thought there was a third but I can't remember now.

**Comment by BHughes3388** [comment]
Both issues have been stated before. There is a work around for the first issue to just build in a close button on the navigation bar. 

The second issue is an easy fix, I just need to do it. 

Problem is the project I developed this for fell through and I haven't even glanced at the code since I wrote it. And I wrote it when I was just beginning to write code and it really needs to be completely rebuilt. 

Still works better and much easier to use than any other youtube oauth library.

**Comment by SteveLobdell** [comment]
Yeah for sure, still awesome I didn't mean to be critical it saved me a lot of time. Just throwing it out there. As I said easy fixes. I just got it through cocoapods and don't like messing with the code because it'll get overwritten if there's ever an update. Maybe I'll branch this and do a pull request with the changes I've made.

**Comment by BHughes3388** [comment]
Oh no worries on the criticism! And I actually meant to mention you should feel free to branch it and make some improvements and I'll merge it. but be aware the code isn't super elegant lol.
